test/parallel/test-net-server-call-listen-multiple-times.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+// This test checks two cases.
+
+// The first one is about calling a listen method again,
+// after first time is fail (without calling close method).
+
+// The second one is about calling a listen method twice.
+// It should be ELISTENCALLEDTWICE but the first server listen call is working.
+
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+let err1 = false;
+let err2 = false;
+
+function case1() {
+
+  const dummyServer = net.Server();
+  const server1 = net.Server();
```

Comment:
You don't need `server1` and `server2` as different names. They are in different scopes. Just use `server` for both.